e2etest/src/test/java/fk/prof/recorder/e2e/BackendProcess.java

```diff
@@ -0,0 +1,63 @@
+package fk.prof.recorder.e2e;
```

Comment:
this package (fk.prof.recorder) does not seem right for BackendProcess, and other classes following this one